llvm/test/Transforms/EntryExitInstrumenter/pre-inliner-instrumentation.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: opt -passes="default<O1>" -S < %s | FileCheck %s
+; RUN: opt -passes="thinlto-pre-link<O2>" -S < %s | FileCheck %s
+; RUN: opt -passes="thinlto-pre-link<O2>,thinlto<O3>" -S < %s | FileCheck %s
+
+target triple = "x86_64-unknown-linux"
+
+define void @leaf_function() #0 {
```

Comment:
mark this function as `alwaysinline` so testing `-O0` works

Reply:
Done.